src/reconcile/rec_col.c

```diff
@@ -603,6 +606,23 @@ __wt_rec_col_var(
     WT_TIME_WINDOW_INIT(&last.tw);
     last.deleted = false;
 
+    /*
+     * When removing a key due to a tombstone with a durable timestamp of "none", also remove the
```

Comment:
When removing a key due to a tombstone with a durable timestamp of "none"

This not only applies to timestamp none but all globally visible timestamps I think.

Reply:
This is only required for the scenarios of mixed-mode tombstones. For normal scenarios, there is no need to remove the history store contents.

Reply:
This is inconsistent to the row store code. I think we should also check the transaction id as well. Only the timestamp is not enough. It is possible that we have an out of order timestamp that is globally visible and there are contents in the history store that is not globally visible. In this case, we still need to remove all the content in the history store so only check ts none is not enough.

Reply:
we write the tombstone only when it is globally visible. I don't see a need for transaction id check here. That is already taken in the rec_upd_select code flow. The out of order problem exists in the row-store code base also, it is not specific to the column store.